build.sbt

```diff
@@ -163,7 +163,7 @@ val slickV = "3.2.3"
 val hsqldbV = "2.3.4"
 val postgresV = "42.2.5"
 val flywayV = "5.2.4"
-val confluentV = "4.1.2"
+val confluentV = "5.4.1"
```

Comment:
will it introduce newer kafka version? If so, we probably want to have them consistent?

Reply:
Hmm, muszę to sprawdzić ... Ale dobry chwyt!

Reply:
In https://docs.confluent.io/current/schema-registry/changelog.html is no information about kafka version.. So I assume that version 2.2 is alsow supported.